ipv-api/src/main/java/uk/gov/di/authentication/ipv/services/IPVAuthorisationService.java

```diff
@@ -182,7 +183,7 @@ public EncryptedJWT constructRequestJWT(
                         .claim("client_id", configurationService.getIPVAuthorisationClientId())
                         .claim("response_type", ResponseType.CODE.toString())
                         .claim("scope", scope.toString())
-                        .claim("vtr", vtr);
+                        .claim("vtr", vtr.stream().map(LevelOfConfidence::getValue).toList());
```

Comment:
I'm assuming this was labeled badly before

Reply:
Yes, the confusion it caused me was part of my motivation to defer converting to string.